src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineExtensionTests.cs

```diff
@@ -20,35 +20,35 @@ namespace Microsoft.Azure.Commands.Compute.Test.ScenarioTests
     public class VirtualMachineExtensionTests
     {
         [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        //[Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
@EmmaZhu You will need to get buy-in from the test owner before disabling these tests